api/images/models/bounding_box.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+from api.models import BaseModel
+
+
+class BoundingBox(BaseModel):
+    image_analysis = models.ForeignKey("Analysis", blank=False, null=False)
+    x = models.DecimalField(blank=False, null=False)
+    y = models.DecimalField(blank=False, null=False)
+    width = models.DecimalField(blank=False, null=False)
+    height = models.DecimalField(blank=False, null=False)
+    confidence = models.DecimalField(blank=False, null=False)
```

Comment:
You can add an arbitrary maximum digit size
```suggestion
    x = models.DecimalField(blank=False, null=False, max_digits=14)
    y = models.DecimalField(blank=False, null=False, max_digits=14)
    width = models.DecimalField(blank=False, null=False, max_digits=14)
    height = models.DecimalField(blank=False, null=False, max_digits=14)
    confidence = models.DecimalField(blank=False, null=False, max_digits=14)
```